.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Can we add .vs to .gitignore?

Reply:
Thanks! This is a git ask, not a Unity one. By including .vs folder in the .gitignore, this will not be including in the pull request. The .vs file has files relevant to Visual Studio code, which are not required for proper execution of the Unity code.
Let me know if this doesn’t make sense